src/styles/components/_button.scss

```diff
@@ -7,6 +7,8 @@
     background-color: var(--mynah-color-button);
     color: var(--mynah-color-button-reverse);
     box-shadow: var(--mynah-shadow-button);
+    font-family: var(--mynah-font-family);
+    font-size: var(--mynah-font-size-medium) !important;
```

Comment:
One of the tests suites on Linux overrides the button style in some certain case. Since they're OS dependent for elements, it is safe to give `!important` to avoid font-size overrides for button. And IDEs are using their own web renderers, we don't know how they can override.